Gemfile

```diff
@@ -70,6 +70,8 @@ group :test do
   gem 'selenium-webdriver'
   # Easy installation and use of web drivers to run system tests with browsers
   gem 'webdrivers'
+  # Database cleaner
```

Comment:
Please remove this comment, we see in the next line that this a database cleaners

Reply:
done, can i merge it now?